crates/wasi-component-loader/src/extension/api/since_0_15_0/wit/mod.rs

```diff
@@ -0,0 +1,34 @@
+pub mod authorization_types;
+pub mod field_resolver_types;
+pub mod grpc;
+pub mod postgres;
+pub mod resolver_types;
+pub mod schema;
+pub mod selection_set_resolver_types;
+
+wasmtime::component::bindgen!({
+    path: "../grafbase-sdk/wit/since_0_15_0/",
+    world: "sdk",
+    async: true,
+    with: {
+        "grafbase:sdk/directive": crate::extension::api::since_0_9_0::wit::directive,
+        "grafbase:sdk/access-log": crate::extension::api::since_0_9_0::wit::access_log,
+        "grafbase:sdk/nats-client": crate::extension::api::since_0_9_0::wit::nats_client,
+        "grafbase:sdk/http-client": crate::extension::api::since_0_9_0::wit::http_client,
+        "grafbase:sdk/cache": crate::extension::api::since_0_9_0::wit::cache,
+        "grafbase:sdk/error": crate::extension::api::since_0_9_0::wit::error,
+        "grafbase:sdk/grpc/grpc-client": crate::resources::GrpcClient,
```

Comment:
re-use as much as you can from wit 14, grpc didn't change. No need to copy-paste more code than necessary

Reply:
same for schema at first sight

Reply:
done

Reply:
Thank you